README.rst

```diff
@@ -66,10 +66,30 @@ pkgdb2::
 
     (my-pkgdb2-env)$ python createdb.py
 
+Setting up PostgreSQL
+=====================
+You can set up PostgreSQL instead of sqlite
+
+For setting up the PostgreSQL database you can look into the `documentation
```

Comment:
You should precise what documentation it is here: ``the `Fedora documentation about PostgresQL<http://...``